cmake/PrivateSdlFunctions.cmake

```diff
@@ -258,12 +258,16 @@ function(target_get_dynamic_library DEST TARGET)
     set(${DEST} ${result} PARENT_SCOPE)
 endfunction()
 
-macro(sdl_check_project_in_subfolder relative_subfolder name vendored_option)
-    if(NOT EXISTS "${PROJECT_SOURCE_DIR}/${relative_subfolder}/CMakeLists.txt")
-        message(FATAL_ERROR "No cmake project for ${name} found in ${relative_subfolder}.\n"
+function(sdl_check_project_in_subfolder relative_subfolder name vendored_option)
+    cmake_parse_arguments(ARG "" "FILE" "" ${ARGN})
+    if(NOT ARG_FILE)
+        set(ARG_FILE "CMakeLists.txt")
```

Comment:
Fyi, this never-used  `FILE` argument is a remainer of an attempt to directly use the meson project.

Reply:
Revert the changes to PrivateSdlFunctions.cmake ?

Reply:
Not needed, just an fyi :)

Reply:
Well, I went ahead and reverted it